public/assets/js/scripts.js

```diff
@@ -322,101 +333,117 @@ $(document).ready(function () {
     const spreadPrivate = initial >= (avgPropertyCost * 0.025) && (avgPropertyCost <= 45000000) && balance / periodic <= 24 / frequency;
     const hybrid = initial >= (avgPropertyCost * 0.01) && balance / periodic <= 24 / frequency;
     
-
-    if (outrightPersonal || outrightMortgage) {
-      let package = {
-        title: `Outright <img src="./assets/img/icons/question-mark.svg" alt="payment">`,
+    const categories = {
+      outrightPersonal: {
+        title: `Outright`,
         message: 'Start BALLing with an initial deposit of 50%'
```

Comment:
Can you put only the value here instead of the whole message? `Start BALLing with an initial deposit of` is common to all the messages and could be included in the message instead of the object. If we want to change this message in the future, I will prefer to change it in one place instead of multiple places (as this is easily prone to typos).